net-misc/snarf/snarf-7.0-r5.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+inherit autotools toolchain-funcs
+
+DESCRIPTION="Small and fast CLI resource grabber for http, gopher, finger, ftp"
+HOMEPAGE="https://www.xach.com/snarf/"
+SRC_URI="https://www.xach.com/snarf/${P}.tar.gz"
+
+KEYWORDS="~alpha ~amd64 ~ppc ~sparc ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+
+PATCHES=(
+	"${FILESDIR}"/snarf-basename-patch.diff
+	"${FILESDIR}"/snarf-unlink-empty.diff
+	"${FILESDIR}"/snarf-fix-off-by-ones.diff
+	"${FILESDIR}"/snarf-fix-build-for-clang16.patch
+)
+
+src_prepare() {
+	eautoconf
+	default
```

Comment:
default must always be _before_ eautoreconf and friends, otherwise user patches can't touch the build system

Reply:
you fixed the other bit but not this?

Reply:
Oh, forgot about this yesterday. I will fix it later. I am not on a gentoo machine to try out of it still works after my changes